src/shared/telemetries/telemetrysender/global_sender.go

```diff
@@ -95,3 +97,34 @@ func IncrementUniqueCounterNetworkMapper(eventType telemetriesgql.EventType, key
 func IncrementUniqueCounterCredentialsOperator(eventType telemetriesgql.EventType, key string) {
 	incrementCounter(telemetriesgql.ComponentTypeCredentialsOperator, eventType, key)
 }
+
+func IntentsOperatorRunActiveReporter(ctx context.Context) {
+	runActiveComponentReporter(ctx, telemetriesgql.ComponentTypeIntentsOperator)
+}
+
+func NetworkMapperRunActiveReporter(ctx context.Context) {
+	runActiveComponentReporter(ctx, telemetriesgql.ComponentTypeNetworkMapper)
+}
+
+func CredentialsOperatorRunActiveReporter(ctx context.Context) {
+	runActiveComponentReporter(ctx, telemetriesgql.ComponentTypeCredentialsOperator)
+}
+
+func runActiveComponentReporter(ctx context.Context, componentType telemetriesgql.ComponentType) {
+	go func() {
+		reporterTicker := time.NewTicker(2 * time.Minute)
```

Comment:
Consider moving this to a configuration flag. Might be useful to be able to control this interval.